app/Http/Controllers/Frontend/FlightController.php

```diff
@@ -94,7 +97,7 @@ public function bids(Request $request)
         $saved_flights = $flights->pluck('id')->toArray();
 
         return view('flights.index', [
-            'title'    => 'Bids',
```

Comment:
Leave translations out until I fix the other commit. I'm rearranging some of the files.